src/components/activity/ActivityList.js

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+function ActivityList({activities, onHandleDelete}) {
+  const handleDelete = event => {
+    const activityId = event.target.dataset.id;
+    onHandleDelete(activityId);
+  };
+  return (
+    <>
+      {activities.map(activity => (
+        <div key={activity.id}>
+          <StyledActivitiList>
+            <p>{activity.activity}</p>
+            <p>{activity.startTime}</p>
+            <p>{activity.endTime}</p>
+            <StyledButton onClick={handleDelete} data-id={activity.id}>
+              Eintrag löschen
+            </StyledButton>
+          </StyledActivitiList>
+        </div>
+      ))}
+    </>
```

Comment:
You should work again here to make sure your semantics are correct. You call it a list after all, but it consists of div.
Maybe like this:

```suggestion
    <StyledActivitiList>
      {activities.map(activity => (
        <li key={activity.id}>
            <p>{activity.activity}</p>
            <p>{activity.startTime}</p>
            <p>{activity.endTime}</p>
            <StyledButton onClick={handleDelete} data-id={activity.id}>
              Eintrag löschen
            </StyledButton>
        </li>
      ))}
    </StyledActivitiList>
```